packages/backend/src/plugins/DemoEventBasedEntityProvider.ts

```diff
@@ -20,9 +20,10 @@ import {
 } from '@backstage/plugin-catalog-node';
 import { EventParams, EventSubscriber } from '@backstage/plugin-events-node';
 import { Logger } from 'winston';
-import { PluginEnvironment } from '../types';
 
-class DemoEventBasedEntityProvider implements EntityProvider, EventSubscriber {
+export class DemoEventBasedEntityProvider
```

Comment:
I feel like this provider should be somewhere else maybe in another plugin. Thoughts?

Reply:
We might want to consider removing it entirely considering the new setup via catalog.ts vs the catalogEventBasedEntityProvider.ts which wouldn't really have made sense without an example. 🤔 

Also, we have a couple of implementations which can be looked at if examples are needed.

Reply:
Ill leave it here for the moment and we can remove in a later refactor. If that is ok?

Reply:
fine with me